forge-gui/res/cardsfolder/upcoming/krenkos_buzzcrusher.txt

```diff
@@ -0,0 +1,11 @@
+Name:Krenko's Buzzcrusher
+ManaCost:2 R R
+Types:Artifact Creature Insect Thopter
+PT:4/4
+K:Flying
+K:Trample
+T: Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ DBDestroy | TriggerDescription$ When CARDNAME enters the battlefield, for each player, destroy up to one nonbasic land that player controls. For each land destroyed this way, its controller may search their library for a basic land card, put it onto the battlefield tapped, then shuffle.
+SVar:DBDestroy:DB$ Destroy | ValidTgts$ Land.nonBasic | TgtPrompt$ Select up to one nonbasic land each player controls | TargetMin$ 0 | TargetMax$ OneEach | TargetsWithDifferentControllers$ True | RememberLKI$ True | SubAbility$ DBSearch
```

Comment:
How do I make it not target?

Reply:
yea, pretty much like you described, no faster way